distribution/lib/Standard/Base/0.0.0-dev/src/Errors/Common.enso

```diff
@@ -197,13 +197,8 @@ type No_Such_Method
        Convert the No_Such_Method error to a human-readable format.
     to_display_text : Text
     to_display_text self =
-        target_name =
```

Comment:
If this change is the culprit of `No_Such_Method` error, can we:
- return the faulty code
- wrap it with `Panic.catch`
- dump some info (Java stack, state of node we are at, etc.?)